module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -646,18 +646,16 @@ public function patronLogin($username, $password)
         if (200 !== $result['code']) {
             throw new ILSException('Problem with Koha REST API.');
         }
-
-        return [
-            'id' => $data['patron_id'],
-            'firstname' => $data['firstname'],
-            'lastname' => $data['surname'],
-            'cat_username' => $username,
-            'cat_password' => (string)$password,
-            'email' => $data['email'],
-            'major' => null,
-            'college' => null,
-            'home_library' => $data['library_id'],
-        ];
+        $patron = $this->createPatronArray(
+            id: $data['patron_id'],
+            cat_username: $username,
+            cat_password: (string)$password,
+            firstname:  $data['firstname'],
+            lastname: $data['surname'],
+            email: $data['email'],
+        );
+        $patron['home_library'] = $data['library_id'];
```

Comment:
It looks like this field is used by the code in [profile.phtml](https://github.com/vufind-org/vufind/blob/9901c04b089daebeaa4abf6693ccce6f45fd56c0/themes/bootstrap5/templates/myresearch/profile.phtml#L43) (where it comes from getMyProfile, rather than from patronLogin). However, it is not documented in the [ILS driver spec](https://vufind.org/wiki/development:plugins:ils_drivers). We should probably fix the documentation!

Reply:
I have updated the spec to match the reality of the code... but should we now move this home_library value to getMyProfile instead of patronLogin here? Does this actually work correctly/as expected in your local environment?